rest-api-spec/src/main/resources/rest-api-spec/test/create/60_refresh.yml

```diff
@@ -8,6 +8,9 @@
               settings:
                   index.refresh_interval: -1
                   number_of_replicas: 0
+                  # make sure that we don't have shard relocations; otherwise search can be executed by a relocated shard,
+                  # which does not have the latest data.
```

Comment:
I'm confused by the comment here. The issue is that the relocated shard issues a refresh at the end of recovery. It has the latest data, the problem is just that it made that data visible.

Reply:
Adjusted in https://github.com/elastic/elasticsearch/pull/69095/commits/bbdeb17ad0af3b0e36a62d9c344a5518a8724aca